azurelinuxagent/ga/update.py

```diff
@@ -1013,13 +1017,10 @@ def _send_heartbeat_telemetry(self, protocol):
         if datetime.utcnow() >= (self._last_telemetry_heartbeat + UpdateHandler.TELEMETRY_HEARTBEAT_PERIOD):
             dropped_packets = self.osutil.get_firewall_dropped_packets(protocol.get_endpoint())
             auto_update_enabled = 1 if conf.get_autoupdate_enabled() else 0
-            # Include vm architecture in the heartbeat message because the kusto table does not have
-            # a separate column for it.
-            vmarch = self._get_vm_arch()
```

Comment:
after removing this, self._get_vm_arch doesn't seem to be used anymore

Reply:
It's used in the run method on line 318

Reply:
I moved it to osutil since we are already using it in update and event